history.txt

```diff
@@ -1,6 +1,39 @@
 OMERO version history
 =====================
 
+5.1.4 (September 2015)
+----------------------
+
+A bug-fix release covering all components. Improvements include:
+
+-  channel buttons fixed in OMERO.web
```

Comment:
Add improvements for moving data between groups in OMERO.web e.g. warnings about tags you will lose